packages/typescript-sdk/.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
```

Comment:
As long as we have both flow and ts, we need to have "custom" eslint configs in the TS packages. Eventually, we will have only one config once flow has been removed.